tests/components/homekit_controller/test_media_player.py

```diff
@@ -58,6 +76,15 @@ def create_tv_service_with_target_media_state(accessory):
     assert state.state == "idle"
 
 
+async def test_tv_read_sources(hass, utcnow):
+    """Test that we can read the input source of a HomeKit TV."""
+    helper = await setup_test_component(hass, create_tv_service)
+
+    state = await helper.poll_and_get_state()
+    assert state.attributes["source"] == "HDMI 1"
+    assert state.attributes["source_list"] == ["HDMI 1", "HDMI 2"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5134240)